Makefile.Common

```diff
@@ -68,6 +68,7 @@ CROSSLINK           := $(TOOLS_BIN_DIR)/crosslink
 GOJUNIT             := $(TOOLS_BIN_DIR)/go-junit-report
 BUILDER             := $(TOOLS_BIN_DIR)/builder
 GOVULNCHECK         := $(TOOLS_BIN_DIR)/govulncheck
+GCI                 := $(TOOLS_BIN_DIR)/gci
```

Comment:
I believe we need to add this to `internal/tools`

Reply:
It already is there, as a golangcilint linter.